src/app/pages/system/CloudCredentials/cloudcredentials-form/cloudcredentials-form.component.ts

```diff
@@ -144,6 +175,13 @@ export class CloudCredentialsFormComponent {
       type: 'input',
       name: 'key-B2',
       placeholder: T('Application Key'),
+      tooltip: T('Backblaze B2 Bucket Application Key. This is\
+                  generated after logging into a B2 account and going to\
```

Comment:
Use present tense: "Backblaze B2 Bucket Application Key. Log in to the B2 account and generate the key on the Buckets page."